AUTHORS

```diff
@@ -3,6 +3,7 @@ intel Linux graphics/media team working on the CMRT:
 Daniel Charles
 Sean V Kelley
 Sameer Kibey
+Midhun Kodiyath
```

Comment:
@MidhunKodiyath due to all patches in patch set include the proper sign-off already